SU-SOE/soe_basic
# D8CORE-3004: Fixing the cards for Events and News to match

# READY FOR REVIEW

# Summary
- Fix ups to have the new cards match SOE theme

# Review By (Date)
- 11/19

# Urgency
- Med

# Steps to Test

1. Pull in this request
2. Create content with new paragraph types.
3. Verify the new cards match the style.

# Affected Projects or Products
- SOE

# Associated Issues and/or People
- D8CORE-3004

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
